distributed/preloading.py

```diff
@@ -119,13 +119,14 @@ def _import_module(name, file_dir=None) -> ModuleType:
     return module
 
 
-async def _download_module(url: str) -> ModuleType:
+def _download_module(url: str) -> ModuleType:
```

Comment:
Apologies for being late to the party. Could you elaborate a bit more on why this change was needed? My guess is there was a use case where we wanted the preload to be downloaded before starting up a worker or nanny. But I personally would find a little more explanation for this change useful

Reply:
Preloads have actions that happen on ...

1.  Import
2. Setup
3. Teardown

*Except* if the preload is a web preload, in which case the import happens at start.  It's an odd exception that this PR removes.  

For the practical use case this has to do with workers asking another service what the scheduler address is when they start up.

Reply:
Thanks Matt for the quick reply. I definitely need to do a better job adding context to my PRs ☹️. I apologize @jrbourbeau that's my fault.